tekton/src/tasks/dependencies/cp4d-service-update.yml.j2

```diff
@@ -79,6 +79,6 @@ spec:
       command:
         - /opt/app-root/src/run-role.sh
         - cp4d_service
-      image: quay.io/ibmmas/cli:latest
+      image: quay.io/ibmmas/cli:11.3.0-pre.mascore3901
```

Comment:
11.3.0-pre.mascore3901 needs to be removed

Reply:
done